* berkely db check is completely broken, but mess with it just enough
  such that it works for Gentoo Prefix

--- configure.ac
+++ configure.ac
@@ -891,7 +891,7 @@
     then
         if test x$ac_bdb_prefix = xyes || test x$ac_bdb_prefix = x
         then
-          ac_bdb_prefix="$mutt_cv_prefix /opt/csw/bdb4 /opt /usr/local /usr"
+          ac_bdb_prefix="$mutt_cv_prefix"
         fi
         for d in $ac_bdb_prefix; do
             bdbpfx="$bdbpfx $d"
@@ -899,7 +899,7 @@
                 bdbpfx="$bdbpfx $d/$v"
             done
         done
-        BDB_VERSIONS="db-4 db4 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 db ''"
+        BDB_VERSIONS="db db-4 db4 db-4.6 db4.6 db46 db-4.5 db4.5 db45 db-4.4 db4.4 db44 db-4.3 db4.3 db43 db-4.2 db4.2 db42 db-4.1 db4.1 db41 ''"
         AC_MSG_CHECKING([for BerkeleyDB > 4.0])
         for d in $bdbpfx; do
             BDB_INCLUDE_DIR=""
